### DISCUSSION TITLE
本周误杀

### DISCUSSION THREAD

**Discussion by cxw620** [Potterli20/hosts#15] [General]
# 误杀
iknowstatic.jomodns.com
login.microsoftonline.com
graph.microsoft.com

**Comment by Potterli20** [comment]
okk我会在github.com/potterli20/file更新

**Comment by cxw620** [comment]
@@||giscus.app^$important
@@||grpc.biliapi.net^$important
@@||*.alidns.com^$important
@@||store-drcn.hispace.dbankcloud.cn^$important
@@||appimg.dbankcdn.com^$important
@@||lf3-cdn-tos.bytescm.com^$important
@@||officeclient.microsoft.com^$important
@@||odc.officeapps.live.com^$important
@@||www.aqours.today^$important
@@||himg.bdimg.com^$important
@@||cdn.jsdelivr.net^$important

**Reply by Potterli20** [reply]
*.alidns.com和f3-tos那个是什么原因，还有bilibili那个，有相关的链接最好发给我

**Reply by cxw620** [reply]
> *.alidns.com和f3-tos那个是什么原因，还有bilibili那个，有相关的链接最好发给我

grpc.biliapi.net 是哔哩哔哩客户端gRPC API, 不能杀
alidns.com这个的阿里云DNS的官网
f3这个是许多字节系网站的静态资源CDN之一

**Reply by Potterli20** [reply]
grpc.biliapi.net：https://github.com/app2smile/rules/issues/64#issuecomment-1225074683
这个嘛，可以屏蔽可以不屏蔽，但是我选择屏蔽好的（我还没有发现有什么影响）
